game/world/world.cpp

```diff
@@ -690,12 +690,16 @@ Bullet& World::shootBullet(const Item &itm, const Npc &npc, const Npc *target, c
   return b;
   }
 
+void World::sendPassivePerc(Npc &self, Npc &other, Item& item, int32_t perc) {
+  wobj.sendPassivePerc(self,other,nullptr,&item,perc);
+  }
+
 void World::sendPassivePerc(Npc &self, Npc &other, Npc &victum, int32_t perc) {
-  wobj.sendPassivePerc(self,other,victum,nullptr,perc);
+  wobj.sendPassivePerc(self,other,&victum,nullptr,perc);
```

Comment:
Since original assumption about `victim` cannot be null is now know to be not valid, we also need to update code that call to his function, such as:
```
// Interactive::implTick
npc.world().sendPassivePerc(npc,npc,npc,PERC_ASSESSUSEMOB);


```

Reply:
Hopefully I caught all of them now.

One interesting decision I had a hard time with, was wether or not to have the World::sendPassivePerc() variants.
Before the victim change, there were two sendPassivePerc variants, hiding the the possible item nullptr from users.

I've gone now with introducing similar variants for victim and victimless variants, but this already increases variants to 4.

The other option would be of course to just add the nullptr when calling World::sentPassivePerc(). I'll defer that to your preference :-) .